form.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
```

Comment:
One of you has your tab-spaces set to 4, and that should be changed to 2, so we don't get this extended indentation in your HTML.